permissions_validators/Cargo.toml

```diff
@@ -10,6 +10,7 @@ edition = "2021"
 iroha_core = { version = "=2.0.0-pre-rc.3", path = "../core", default-features = false }
 iroha_data_model = { version = "=2.0.0-pre-rc.3", path = "../data_model", default-features = false }
 iroha_macro = { version = "=2.0.0-pre-rc.3", path = "../macro" }
+iroha_schema = { version = "=2.0.0-pre-rc.3", path = "../schema" }
```

Comment:
I don't think this is necessary since specialized tokens are converted to generic on the client side

Reply:
I would love if they weren't converted and were exchanged with Iroha in their specialized form

Reply:
Cli tool relies on schema to dump the tokens, that's why I used it.